### PR TITLE
Verbs: Misc items

### PR DESCRIPTION
**Pull request by yishaih**
This series includes a fix in some udaddy bad flow.
In addition it cleans-up some unused stuff in the verbs area related to 'fw_ver'.

### CODE REVIEW

**Comment by galpress** [comment]
Patrisious is missing a last name.

**Comment by yishaih** [comment]
Thanks Gal, just added.

File: libibverbs/libibverbs.map.in [hunk 1]
```diff
@@ -215,7 +215,6 @@ IBVERBS_PRIVATE_@IBVERBS_PABI_VERSION@ {
 		ibv_cmd_req_notify_cq;
 		ibv_cmd_rereg_mr;
 		ibv_cmd_resize_cq;
-		ibv_get_fw_ver;
```
Comment (rleon) [inline libibverbs/libibverbs.map.in:218]: Don't you need to change IBVERBS_PABI_VERSION too?

Comment (jgunthorpe) [reply libibverbs/libibverbs.map.in:218]: Yes.. And we really should have a CI test for this, I just don't know how to write it.

**Comment by rleon** [comment]
We can use abi checker from our stable@. The one that Nicolas is using.

File: libibverbs/libibverbs.map.in (hunk 1 above)
Comment (rleon) [reply libibverbs/libibverbs.map.in:218]: ABI dumper like we have in stable@?

Comment (yishaih) [reply libibverbs/libibverbs.map.in:218]: I have bumped the private version to be 33 as of current library one, a CI can be really helpful here.

**Review by rleon — APPROVED** [review]

**Pull request merged**